frontend/src/components/APIService.js

```diff
@@ -0,0 +1,71 @@
+export default class APIService {
+  static LoginUser(body) {
+    return fetch('/api/token/login/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(body)
+    }).then((resp) => resp.json());
+  }
+
+  static RegisterUser(body) {
+    return fetch('/api/users/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(body)
+    }).then((resp) => resp.json());
+  }
+
+  static LogoutUser(cookie) {
+    console.log(cookie);
+    return fetch('/api/token/logout/', {
+      method: 'POST',
+      headers: {
+        Authorization: `Token ${cookie}`
+      }
+    });
+  }
+
+  static messageCreation(body, token) {
```

Comment:
same like one of the previous comments, `body` doesn't say what exactly it is
so rather than passing body you can destructure here it to `({ room, body })` (but here there is still body xd)